arbnode/node.go

```diff
@@ -203,7 +209,7 @@ func ConfigDefaultL1NonSequencerTest() *Config {
 	config.SeqCoordinator.Enable = false
 	config.BlockValidator = staker.TestBlockValidatorConfig
 	config.SyncMonitor = TestSyncMonitorConfig
-	config.Staker = staker.TestL1ValidatorConfig
+	config.Staker = legacystaker.TestL1ValidatorConfig
 	config.Staker.Enable = false
```

Comment:
set BOLD config as well?

Reply:
The `boldstaker.DefaultBoldConfig` is included in the `ConfigDefault` which is used to initialize the local `config` object. So, I think it's already included. And, by default, the bold config includes `Enabled = false`. So, I don't think we really need to change anything like we do in the `legacystaker`.

Marking as resolved.